site/en/blog/meet-the-new-css-color-spaces/index.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: layouts/blog-post.njk
+title: Meet the new CSS color spaces
+description: >
+  CSS Color 4 brings wide gamut color tools and capabilities to the web: more colors, manipulation functions, and better gradients.
+subhead: >
+  CSS Color 4 brings wide gamut color tools and capabilities to the web: more colors, manipulation functions, and better gradients.
+date: 2023-02-01
+authors:
+  - argyle
+tags:
+  - css
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/1EFisnQsVHucCNJdjQvy.png
+alt: >
+  A moment from The Wizard of Oz movie and half of it is in black and white and the other half is in color.
```

Comment:
I think you are using a different image for the blog

Reply:
got it thx!